portfolio/src/test/java/com/google/sps/HashTableTest.java

```diff
@@ -39,60 +41,66 @@ public void noComments() {
   @Test
   public void oneBlogPostForEachTag() {
     //Return a Hashmap with LinkedList of size one for each tag.
-    BlogHashMap<String, LinkedList<BlogMessage>> actual = new BlogHashMap<>();
-    actual("#Pop") = BlogMessage(1, "#Pop", "I enjoy pop music", "", "Steven", new ArrayList<String>(), System.currentTimeMillis());
-    actual("#Country") = BlogMessage(2, "#Country", "Country music is the best music", "", "Tayyaba", new ArrayList<String>(), System.currentTimeMillis());
-    actual("#Classical") = BlogMessage(3, "#Classical", "Classical music makes you smarter", "", "Andrew", new ArrayList<String>(), System.currentTimeMillis());
+    BlogHashMap actual = new BlogHashMap();
+    List<BlogMessage> temp = new ArrayList<>();
+    temp.add(new BlogMessage((long) 1, "#general", "I enjoy pop music", "", "Steven", new ArrayList<String>(), System.currentTimeMillis()));
+    temp.add(new BlogMessage((long) 2, "#wellbeing", "Country music is the best music", "", "Tayyaba", new ArrayList<String>(), System.currentTimeMillis()));
+    temp.add(new BlogMessage((long) 3, "#music", "Classical music makes you smarter", "", "Andrew", new ArrayList<String>(), System.currentTimeMillis()));
+    actual.putInMap(temp);
 
-    Set<String> keys = actual.keySet();
+    Collection<String> keys = Arrays.asList("#general", "#wellbeing", "#music");
     for(String key : keys){
-      Assert.assertEquals(1, actual.get(key).size());
+      Assert.assertEquals(1, actual.getMessages(key).size());
```

Comment:
About checking the equality of two objects x & y, if you define a equals() method in the object's class, you can just use Assert.assertEquals(x, y);

This would be more convenient, expressive, and concise.